src/networks.json

```diff
@@ -1247,5 +1247,31 @@
     ],
     "explorer": "https://baobab.scope.klaytn.com/",
     "logo": "ipfs://QmYACyZcidcFtMo4Uf9H6ZKUxTP2TQPjGzNjcUjqYa64dt"
-    }
+    },
+    "1002": {
+    "key": "1002",
+    "name": "KardiaChain Mainnet",
+    "shortName": "KAI",
+    "chainId": 24,
+    "network": "mainnet",
+    "multicall": "0xd9c92F2287B7802A37eC9BEce96Aa65fb5f31E1b",
+    "rpc": [
+      "https://kai-internal.kardiachain.io"
+    ],
+    "explorer": "https://explorer.kardiachain.io",
+    "logo": "ipfs://QmVH3uyPQDcrPC1DMUWCb7HayMv1oMAiKehuWwP2C2fdgM"
+    },
+    "1003": {
+      "key": "1003",
+      "name": "KardiaChain Testnet",
+      "shortName": "KAI",
+      "chainId": 242,
+      "network": "testnet",
+      "multicall": "0x55C2f041dBf8eB1120fecd3B12580a424487bec1",
```

Comment:
How to read the contract code of this 
https://explorer-dev.kardiachain.io/address/0x55C2f041dBf8eB1120fecd3B12580a424487bec1

Reply:
Hi @ChaituVR Our testnet explorer UI is older and not have a clear interface for verified smc. Checking with our team, testnet for Snapshot is not important so I'll delete the testnet config